.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: "🛠 goal: fix, 🚦 status: awaiting triage, 💻 aspect: code, 🟧 priority: high"
+title: "[Bug] <Replace this with actual title>"
+---
+
+## Description
+<!-- Concisely describe the bug. -->
+
+## Reproduction
+<!-- Provide detailed steps to reproduce the bug -->
+1. <!-- Step 1 ... -->
+2. <!-- Step 2 ... -->
+3. <!-- Step 3 ... -->
+4. See error.
+
+## Expectation
+<!-- Concisely describe what you expected to happen. -->
+
+## Screenshots
+<!-- Add screenshots to show the problem; or delete the section entirely. -->
+
+## Environment
+<!-- Please complete this, unless you are certain the problem is not environment specific. -->
+ - Device: (_eg._ iPhone Xs; laptop)
+ - OS: (_eg._ iOS 13.5; Fedora 32)
+ - Browser: (_eg._ Safari; Firefox)
+ - Version: (_eg._ 13; 73)
+ - Other info: (_eg._ display resolution, ease-of-access settings)
+
+## Additional context
+<!-- Add any other context about the problem here; or delete the section entirely. -->
+
+## Resolution
+<!-- Replace the [ ] with [x] to check the box. -->
+- [ ] I would be interested in resolving this bug.
```

Comment:
Let's remove this checkbox for a few reasons,

1. it tends to confuse users (we get a lot of people incorrectly checking the box such as `[ x]` or `[x ]`
2. it misuses [GitHub task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists)
3. in general, we shouldn't require too much ceremony when people create issues

```suggestion
```